build.sbt

```diff
@@ -21,6 +21,11 @@ ThisBuild / githubWorkflowBuild := Seq(
   WorkflowStep.Sbt(List("clean", "coverage", "test"), name = Some("Build project"))
 )
 
+ThisBuild / githubWorkflowJavaVersions := Seq(
+  // See https://github.com/sbt/sbt-github-actions#jdk-settings
+  JavaSpec.temurin("11")
```

Comment:
This was necessary to preserve _status-quo_ since the default value is Java 8. We can extend this sequence to test against multiple java versions, i.e. 11 and 17.